dev-infrastructure/modules/aks-cluster-base.bicep

```diff
@@ -218,6 +243,40 @@ resource aksClusterAdminRoleAssignment 'Microsoft.Authorization/roleAssignments@
   }
 }
 
+var istioIngressGatewayIPAddressName = 'aro-hcp-istio-ingress'
```

Comment:
nit: this value is also used in the `istio.sh` script.
do you think it would make sense to have it in config.yaml and feed it into both places?

Reply:
I originally had it and then removed it before pushing my PR. The main (only?) benefit is that there is no hardcoding, but the flipside is this requires quite a bit of plumbing for little benefits IMO. I would not expect us to ever have to change this name but maybe there is a use case I am not thinking about?

Added in d3562a3 - PTAL

Reply:
i understand your concern. thanks for adding it anyways for the time being.
we can discuss in a team meeting how we want to do things like these in the future and revamp.